Modules/termios.c

```diff
@@ -232,13 +240,19 @@ termios_tcsetattr_impl(PyObject *module, int fd, int when, PyObject *term)
 
     int i;
     PyObject *v;
+    long l;
```

Comment:
Nit: in general I try to avoid `l` as a variable name, since it can look very similar to `1` and `I`, depending the font.